app/assets/javascripts/fe/fe.public.nojquery.js

```diff
@@ -310,7 +314,8 @@
 
       if(  payments_made)
         {
-          this.savePage($('#' + fe.pageHandler.current_page));  // in case any input fields on submit_page
```

Comment:
I should note that this wasn't working, at least not reliably for all apps using FE